src/Simplex/Chat.hs

```diff
@@ -5315,6 +5365,29 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
     checkHostRole GroupMember {memberRole, localDisplayName} memRole =
       when (memberRole < GRAdmin || memberRole < memRole) $ throwChatError (CEGroupContactRole localDisplayName)
 
+    xGrpMemBlock :: GroupInfo -> GroupMember -> MemberId -> Bool -> RcvMessage -> UTCTime -> m ()
+    xGrpMemBlock gInfo@GroupInfo {groupId, membership} m@GroupMember {memberRole = senderRole} memId blocked msg brokerTs
+      | membershipMemId == memId =
+          -- member shouldn't receive this message about themselves
+          messageError "x.grp.mem.block: admin blocks you"
```

Comment:
this would show in terminal, which is probably not something we want? although may be ok as low priority error that will be hidden by default...

Reply:
this shouldn't happen, it may also indicate a bug to us